lxd/db.go

```diff
@@ -12,6 +12,11 @@ import (
 	_ "github.com/mattn/go-sqlite3"
 )
 
+var (
+	DbErrAlreadyDefined = fmt.Errorf("already exists")
+	NoSuchImageError    = errors.New("No such image")
```

Comment:
Yeah, that's what I was thinking. It looks like fmt.Errorf is mildly more expressive, so perhaps we should go with that everywhere just to reduce cognitive load. Anyway, if it's on your TODO, that's cool with me.